experiment/simulation/index.html

```diff
@@ -1,20 +1,269 @@
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/jsPlumb/2.15.6/js/jsplumb.min.js"></script>
+	<link rel="stylesheet" href="./css/main.css">
 </head>
+
 <body>
-		<!-- Your code goes here-->
+	<div class="columns mt-2 is-variable is-1 is-centered is-flex-wrap-wrap">
+		<div class="v-tabs">
+			<ul>
+				<li class="is-active">
+					<a>
+						<span>LATCH</span>
+					</a>
+				</li>
+				<li>
+					<a>
+						<span>FLIPFLOP</span>
+					</a>
+				</li>
 
-    <p><li>Make sure that you have<a href="../../../Introduction.html" target="_PARENT"> prerequisites</a> installed to run experiment</li> </p>
+			</ul>
+		</div>
+	</div>
 
-   - <a href="exp6/exp6_latch_positive.html">Start the Positive Level D-Latch and D-FlipFlop Experiment</a> <br/> 
+	<div class="columns is-centered">
+		<div class="column">
+			<div class="v-collapsible-instruction-container v-instruction-container">
+				<div class="v-collapsible is-size-5" id="task-description">
+					Instructions
+				</div>
+				<div class="v-content" style="text-align: left; padding-left: 10%; padding-right: 10%;">
+					<ul style="list-style: disc;">
+						<li>Click on components on the left side to add them to the circuit board.</li>
+						<li>Drag and drop components to move them around, connect them using wires by dragging your
+							cursor from one endpoint to another.</li>
+						<li>Double click on a wire to delete it.</li>
+						<li>Double click or long press on input element to change it's value</li>
+						<li>Select the appropriate tab and click on validate after completion of circuit</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+	</div>
 
-   - <a href="exp6/exp6_negative/exp6_flipflop_negative.html">Start the Negative Edge D-FlipFlop Experiment</a>
+	<div class="columns is-variable is-1 is-centered is-flex-wrap-wrap">
+		<div id="start" class="column is-2-desktop is-2-tablet is-3-mobile center-content">
+			<button class="v-button" onclick="circuitValidate()">Validate</button>
+		</div>
+		<div id="restart" class="column is-2-desktop is-2-tablet is-3-mobile center-content">
+			<button class="v-button" onclick="location.reload()">Reset</button>
+		</div>
+	</div>
 
+	<div class="columns is-variable is-1 is-centered is-flex-wrap-wrap">
+		<div class="column is-2-desktop is-3-tablet is-2-mobile center-content">
+			<div class="v-datalist-container">
+				<div class="v-datalist-title">
+					ToolBar
+				</div>
+				<div class="v-datalist-row toolbar-item-list">
+					<div class="toolbar-component" onclick="compMux()">
+						<div class="each-toolbar-component has-text-centered">
+							<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+								version="1.1" viewBox="-0.5 10 124 143">
+								<g class="demo-transistor">
+									<rect x="21" y="21" width="80" height="120" />
+									<g transform="translate(-0.5 -0.5)">
+										<switch>
+											<foreignObject pointer-events="none" width="100%" height="100%"
```

Comment:
Why are we using foreignObject?

Reply:
It is needed otherwise the text showing the components name is not rendered properly.